cmd/vocli/commands/keys.go

```diff
@@ -0,0 +1,267 @@
+package commands
+
+import (
+	"bufio"
+	"crypto/rand"
+	"encoding/hex"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	ethkeystore "github.com/ethereum/go-ethereum/accounts/keystore"
+	"github.com/ethereum/go-ethereum/cmd/utils"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/google/uuid"
+	"github.com/spf13/cobra"
+	"go.vocdoni.io/dvote/client"
+	"go.vocdoni.io/dvote/crypto/ethereum"
+	"go.vocdoni.io/proto/build/go/models"
+	"golang.org/x/term"
+	"google.golang.org/protobuf/proto"
+)
+
+const (
+	scryptN = ethkeystore.StandardScryptN
+	scryptP = ethkeystore.StandardScryptP
+	keyExt  = ".vokey"
+)
+
+var keysCmd = &cobra.Command{
+	Use:   "keys",
+	Short: "Do things related to keys",
+	Long:  "In Vochain, we make the distinction between a key and an account. A Key controls an Account which is stored in the blockchain's state. The Key itself is stored in encrypted form on the user's disk.",
+}
+
+var keysNewCmd = &cobra.Command{
+	Use:   "new [pre-existing keyfile]",
+	Short: "Generate a new key and create its corresponding Account on the chain. Or just create the Account onchain.",
+	Long:  "Generate a new key and send a SetAccountInfo Tx in one step. Or, if you already have a keyfile, send a SetAccountInfo Tx.",
+	Args:  cobra.MaximumNArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		var key *ethkeystore.Key
+		var keyPath string
+
+		fmt.Fprintf(Stdout, "Step 1/2: First we will generate the key and save it on your disk, encrypted.\nUnlike other blockchains, an Account for this key must first be created on the blockchain for others to send you funds (or any other operations).\nOnce the key is generated and saved, we will send a SetAccountInfo transaction to the blockchain in order to create an Account for this key. \nFear not, if you have only generated a key, you may skip the key generation process and directly send the SetAccountInfo transaction by re-running 'keys new <full path to keyfile>'\n\n")
```

Comment:
This line is too long

Reply:
Do you mean i should use `` in Go comments or i should rephrase the help message?

Reply:
Use `` :)